### ISSUE TITLE
'Select' menu item & iPad

### ISSUE THREAD

**Issue by grape6** [johndoh/roundcube-swipe#9]
'Select' menu item in menu box above message list is always grey if swipe plugin enabled. 

iPad should be in horizontal position, 
ios 12.2, safari & firfox.

RCM git, only contextmenu & swipe plugins.

**Comment by johndoh** [comment]
fixed in  86fb57a

**Comment by svatos** [comment]
Is it possible to check if this problem is still persists? If I swipe-select some emails, I can't mark them after from the submenu.

![image](https://user-images.githubusercontent.com/10947341/101782339-29f32380-3af9-11eb-88eb-752feb6f88d8.png)

RC composer:
```
    "require": {
        "php": ">=5.4.0",
        "pear/pear-core-minimal": "~1.10.1",
        "pear/auth_sasl": "~1.1.0",
        "pear/net_idna2": "~0.2.0",
        "pear/mail_mime": "~1.10.0",
        "pear/net_smtp": "~1.8.1",
        "pear/crypt_gpg": "~1.6.3",
        "pear/net_sieve": "~1.4.3",
        "roundcube/plugin-installer": ">=0.2.0",
        "masterminds/html5": "~2.5.0",
        "endroid/qr-code": "~1.6.5",
        "kolab/net_ldap3": "~1.1.1",
        "johndoh/swipe": "^0.3.0"
    },
```
